src/main/resources/db/migration/V1_0__initial_schema.sql

```diff
@@ -1,140 +1,115 @@
 
 create table attachment (
-    category enum ('CONTRACT','OTHER'),
-    extension varchar(255),
-    file varchar(255),
-    id bigint not null auto_increment,
-    mime_type varchar(255),
-    name varchar(255),
-    note varchar(255),
+    id                      bigint not null auto_increment,
+    contract_id             varchar(10) not null,
+    category                enum ('CONTRACT','OTHER'),
+    filename                varchar(255),
+    mime_type               varchar(255),
+    note                    varchar(255),
+    content                 longblob,
     primary key (id)
 ) engine=InnoDB;
 
 create table contract (
-    additional_terms varchar(255),
-    case_id bigint,
-    contract_id varchar(10),
-    description varchar(4096),
-    id bigint not null auto_increment,
-    index_terms varchar(255),
-    municipality_id varchar(255),
-    signed_by_witness bit,
-    status enum ('ACTIVE','DRAFT','TERMINATED'),
-    version integer,
+    id                      bigint not null auto_increment,
+    municipality_id         varchar(255),
+    contract_id             varchar(10),
+    version                 integer,
+    type                    enum ('LAND_LEASE'),
+    status                  enum ('ACTIVE','DRAFT','TERMINATED'),
+    case_id                 bigint,
+    description             varchar(4096),
+    signed_by_witness       bit,
+    additional_terms        json,
+    index_terms             json,
+    extra_parameters        json,
     primary key (id)
 ) engine=InnoDB;
 
-create table contract_attachments (
-    attachments_id bigint not null,
-    contract_entity_id bigint not null
-) engine=InnoDB;
-
-create table contract_extra_parameter (
-    contract_id bigint not null,
-    parameter_key varchar(255) not null,
-    parameter_value varchar(255) not null,
-    primary key (contract_id, parameter_key)
-) engine=InnoDB;
-
-create table contract_stakeholders (
-    stakeholders_id bigint not null,
-    contract_entity_id bigint not null
-) engine=InnoDB;
-
 create table land_lease_contract (
-    area integer,
-    auto_extend bit,
-    end date,
-    lease_duration integer,
-    lease_extension integer,
-    period_of_notice integer,
-    start date,
-    external_reference_id varchar(255),
-    id bigint not null,
-    lease_fees varchar(255),
-    leasehold_description varchar(255),
-    object_identity varchar(255),
-    area_data longblob,
-    invoice_interval enum ('YEARLY','QUARTERLY','MONTHLY'),
-    invoiced_in enum ('ADVANCE','ARREARS'),
-    land_lease_type enum ('LEASEHOLD','USUFRUCT','SITELEASEHOLD'),
-    leasehold_type enum ('AGRICULTURE','APARTMENT','BUILDING','DWELLING','OTHER'),
-    usufruct_type enum ('HUNTING','FISHING','MAINTENANCE','OTHER'),
+    id                      bigint not null,
+    land_lease_type         enum ('LEASEHOLD','USUFRUCT','SITELEASEHOLD'),
+    leasehold_type          enum ('AGRICULTURE','APARTMENT','BUILDING','DWELLING','OTHER'),
```

Comment:
See comment above.

(comment applicable for all enum data types in the db)

Reply:
See above